### PR TITLE
Mailing List Setup

### LINKED ISSUE #121: Create mailing List + Add logic to textfield in static site

### PR DESCRIPTION
**Pull request by cyrusdiego**
# Overview

Integrate Mailchimp to text input.
Used [this](https://dev.to/itnext/setting-up-a-news-letter-with-react-and-mailchimp-2f6p) guide to help setup [this](https://www.npmjs.com/package/react-mailchimp-subscribe) npm package. 

# Changes

- Installed [react-mailchimp-subscribe](https://www.npmjs.com/package/react-mailchimp-subscribe) npm package 
- Able to add email to mailing list 

# Issue tracking

Closes #121 

# Demo 
image
image
image

### CODE REVIEW

File: www/package.json [hunk 1]
```diff
@@ -50,6 +53,7 @@
     },
     "devDependencies": {
         "@types/react-router-dom": "^5.1.7",
+        "@types/validator": "^13.1.4",
```
Comment (calebschoepp) [inline www/package.json:56]: Is there a reason this is in dev deps?

File: www/src/routes/Landing.tsx [hunk 2]
```diff
@@ -242,6 +249,18 @@ const Services: FC = () => {
 const CallToAction: FC = () => {
     const classes = useStyles();
 
+    const getMessage = (status: string | null): string => {
+        if (status == 'error') {
+            return 'Something went wrong :( Please try again';
+        } else if (status == 'sending') {
+            return 'Adding you to the mailing list...';
+        } else if (status == 'success') {
+            return 'You have been added to the mailing list :)';
```
Comment (calebschoepp) [inline www/src/routes/Landing.tsx:258]: Maybe use actual emojis? :smile:

File: www/src/routes/MobileLanding.tsx [hunk 3]
```diff
@@ -38,17 +53,27 @@ const MobileLanding: FC = () => {
...
+                                        valueValidator={(email) => validator.isEmail(email)}
+                                    />
+                                    <Grid item>
+                                        <Typography align='center' variant='body1' style={{ fontWeight: 200 }}>
+                                            {getMessage(status)}
+                                        </Typography>
+                                    </Grid>
+                                </Grid>
+                            )}
+                        />
```
Comment (calebschoepp) [inline www/src/routes/MobileLanding.tsx:76]: Is this identical to the one for the browser page? Could this just be it's own component?

**Review by calebschoepp — APPROVED** [review]
Great work, couple comments but nothing blocking. LGTM!

File: www/package.json (hunk 1 above)
Comment (cyrusdiego) [reply www/package.json:56]: No particular reason, I think that was the dev dependency flag was just set from wherever I copied from

File: www/src/routes/Landing.tsx (hunk 2 above)
Comment (cyrusdiego) [reply www/src/routes/Landing.tsx:258]: True

File: www/src/routes/MobileLanding.tsx (hunk 3 above)
Comment (cyrusdiego) [reply www/src/routes/MobileLanding.tsx:76]: Yeah good point, can do a quick refactor

**Pull request merged**